### ISSUE TITLE
What's the difference between VCF_caller and very_sensitive_caller?

### ISSUE THREAD

**Issue by jumpyknight** [google/deepvariant#256]
The very_sensitive_caller seems to be a new feature in the release 0.9.0, what's the difference between the very sensitive caller and the usual one?

**Comment by AndrewCarroll** [comment]
Hi @jumpyknight 

**tl;dr - VCF_caller represents experimental functionality not ready for production use**

DeepVariant has three stages  **make_examples** identifies candidate positions that may be variants using relatively simple human-written heuristics. **call_variants** applies the trained neural net model to identify which of these candidates are real variants and at what probability. **postprocess_variants** converts these probability to a VCF output.

From the first versions of DeepVariant, very_sensitive_caller has been the logic used to generate candidates for make_examples. VCF_caller is an experimental feature we have been developing that would allow the generation of candidates directly from an input VCF, so that different (or third-party) logic could be applied to generate candidates.

However, this feature is not ready for production use. Its inclusion here occurs because this code is in our main branch at the release time and reflects our internal use and experiments with it. DeepVariant v0.9 still uses very_sensitive_caller to generate candidates and VCF_caller is not used for any released model.

We attempt to fully document and mention in release notes features that are ready for use. Your eyes to the released code are astute, I was not expecting to field questions for VCF_caller.

**Comment by jumpyknight** [comment]
I thought the "very sensitive" has something to do with the indel threshold, but the result turned out to be more exciting!
I just wondered if it's possible for deepvariant to generate candidates according to an existing VCF, and now you have been working for it.
It would be a great feature for uses like optimizing deepvariant models with on-the-flow data, or ensemble calling in somatic-variant detecting.
